astropy/io/fits/tests/test_table.py

```diff
@@ -3035,6 +3035,32 @@ def test_tolist_from_file(self):
             assert hdu.data.tolist() == [[[45, 56], [11, 3]], [[11, 12, 13], [12, 4]]]
             assert hdu.data['var'].tolist() == [[45, 56], [11, 12, 13]]
 
+    @pytest.mark.skipif('sys.maxsize < 2**32' or 'sys.platform == "win32"')
```

Comment:
Apparently I'm unable to understand how the `skipif` works.... sorry for the multiplication of the commits...